src/txmempool.h

```diff
@@ -35,6 +35,10 @@ inline bool AllowFree(double dPriority)
 
 /** Fake height value used in Coin to signify they are only in the memory pool (since 0.8) */
 static const uint32_t MEMPOOL_HEIGHT = 0x7FFFFFFF;
+/** Length of time in seconds over which to smooth the tx rate */
+static const double TRANSACTION_RATE_SMOOTHING_WINDOW_SECONDS = 60;
+/** Sample resolution in milliseconds over which to compute the instantaneous transaction rate */
+static const int INSTANTANEOUS_TRANSACTION_RATE_RESOLUTION_MILLISECONDS = 1000;
 
```

Comment:
nit: seems like a very long variable name, what about just TRANSACTION_RATE_RESOLUTION? . The comment above already says it's in millis

Reply:
Going to shorten names to `TX_RATE_RESOLUTION_MILLIS` and `TX_RATE_SMOOTHING_SEC`.